deepcell/model_zoo/tracking.py

```diff
@@ -351,7 +364,7 @@ def get_appearance_encoder(self):
         x = TimeDistributed(ImageNormalization2D(norm_method='whole_image',
                                                  name='imgnrm_ae'))(x)
 
-        for i in range(self.time_window):
+        for i in range(5):
```

Comment:
Any reason to hardcode 5 rather than leave it as a configurable parameter?

Reply:
Dave’s reasoning is we only want to do 5 convolution and pooling operations and it’s alright to have it hardcoded for now as the crop size (32) is also currently hardcoded.